src/redisjson.rs

```diff
@@ -7,11 +7,13 @@
 use crate::backward;
 use crate::error::Error;
 use crate::formatter::RedisJsonFormatter;
+use crate::json_node::{replace_with, JsonValueUpdater};
 use crate::nodevisitor::{StaticPathElement, StaticPathParser, VisitStatus};
+use crate::select::{Selector, SelectorMut};
 use crate::REDIS_JSON_TYPE_VERSION;
 
 use bson::decode_document;
-use jsonpath_lib::SelectorMut;
+// use index::schema_map; todo: is this needed? it breaks the build!!!
```

Comment:
Looks like a leftover from the old search integration; should be removed.

Reply:
Removed